src/debugging/python/PythonDebugHelper.ts

```diff
@@ -94,7 +94,7 @@ export class PythonDebugHelper implements DebugHelper {
                 removeContainerAfterDebug: debugConfiguration.removeContainerAfterDebug
             },
             debugLauncherPath: debugConfiguration.debugLauncherPath || launcherPath,
-            debugAdapterHost: debugConfiguration.debugAdapterHost || await this.getDebugAdapterHost(context),
+            debugAdapterHost: debugConfiguration.debugAdapterHost || await this.getDebugAdapterHost(context) || '172.17.0.1',
```

Comment:
What is this IP address, why is this one a good default?

Reply:
This is the IP address that represents the host within the container, on the `bridge` network. So far this is the only one I've ever seen--same address on my Windows desktop, Macbook, Linux VM, and multiple customers' Linux machines as well.

Their [docs](https://docs.docker.com/network/network-tutorial-standalone/) explicitly mention that address as well (for `bridge`), but it may be more as an example than a prescriptive statement.

Reply:
Roger. Internet poking suggests 172.17.0.0/16 is the default bridge network address space. I would just put a comment on that in the code then